pgp/keysource.go

```diff
@@ -318,7 +318,7 @@ func (key *MasterKey) encryptWithGnuPG(dataKey []byte) error {
 		fingerprint,
 		"--no-encrypt-to",
 	}
-	err, stdout, stderr := gpgExec(key.gnuPGHome(), args, bytes.NewReader(dataKey))
+	err, stdout, stderr := gpgExec(key.gnuPGHomeDir, args, bytes.NewReader(dataKey))
```

Comment:
Just confirming that this shouldn't be:  `gnuPGHome(key.gnuPGHomeDir)`

Reply:
It shouldn't, because that would re-introduce the behavior of always supplying a home directory (resulting in `--homedir` getting configured).